app/views/hub/users/edit_role.html.erb

```diff
@@ -38,7 +38,7 @@
         <div class="form-group">
           <%= label_tag(:coalition_id, t("devise.invitations.new.coalition_label"), class: "form-question") %>
           <div class="form-width--long">
-            <% coalition_value = @user.role.is_a?(CoalitionLeadRole) && @user.role.coalition.present? ? [{ value: @user.role.coalition.id, name: @user.role.coalition.name }].to_json.to_s.html_safe : [] %>
+            <% coalition_value = @user.role.is_a?(CoalitionLeadRole) && @user.role.coalition.present? ? taggable_format([{ value: @user.role.coalition.id, name: @user.role.coalition.name }]) : [] %>
```

Comment:
Non-blocking:

This ternary returns different data types depending which side is used. Can that be cleaned up?

Specifically, I'm under the impression `taggable_format()` returns a string, but after the `:` we see `[]`, which is an array.

Perhaps it's best to move the ternary into the call to `taggable_format` like

```
taggable_format(
  condition ? [{ value: @user.role.coalition.id, name: @user.role.coalition.name }] : []
)
```

That way we'd get the same data type out no matter what.